client/src/components/DataEntry/MemberForm.tsx

```diff
@@ -50,7 +50,7 @@ const MemberForm: React.FC<IProps> = ({
           values,
           handleSubmit,
         }: FormikProps<[IAuditBoardMember, IAuditBoardMember]>) => (
-          <Form>
+          <form>
```

Comment:
@benadida There's a glitch with types in Formik and React that makes `Form` fail unless you add `translate="true"` to it, which is... kind of a ridiculous solution imo. So since we aren't actually using the magic that `Form` provides, I opted to just swap it out for the regular html version and avoid the issue.